### PR TITLE
Add php nightly

### PR DESCRIPTION
**Pull request by othercorey**

### CODE REVIEW

**Comment by dereuromark** [comment]
You need to run nighly (PHP8) with ignore platform reqs in composer

**Comment by garas** [comment]
> 
> 
> You need to run nighly (PHP8) with ignore platform reqs in composer

Or allow PHPUnit 8 || 9 if compatible.

**Comment by dereuromark** [comment]
No, as your dependencies dont do that properly either ;)
So if you wanna see results, you gotto.

File: .travis.yml [hunk 1]
```diff
@@ -35,7 +39,7 @@ before_install:
       fi
 
 install:
-  - composer install --no-interaction --prefer-dist
+  - composer install --no-interaction --prefer-dist --ignore-platform-reqs
```
Comment (dereuromark) [inline .travis.yml:42]: I usually run --ignore-platform-reqs only for php8 (using a if check)

**Comment by dereuromark** [comment]
Sry I thought you wanted to put the nightly in perminantly (possibly as allow_failures entry).

File: .travis.yml (hunk 1 above)
Comment (dereuromark) [reply .travis.yml:42]: https://github.com/cakephp/cakephp/blob/master/.travis.yml#L59-L60

**Pull request closed without merging**